.github/workflows/main.yaml

```diff
@@ -0,0 +1,96 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
+  # the following triggers are for testing purposes only. remove before merge.
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+env:
+  LANG: en_US.UTF-8
+  LC_ALL: en_US.UTF-8
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      sinkVersion: ${{ steps.lookupSinkVersion.ouputs.result }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
It looks like there are some deprecation warnings so we should update some of the uses actions.

![Screenshot 2023-03-21 at 10 02 50](https://user-images.githubusercontent.com/126863902/226559776-e4d41deb-e7c2-4916-b420-e30dd1f13287.png)

Reply:
These are fixed.